### DISCUSSION TITLE
The public availability of VMware Fusion for Apple Silicon Public Tech Preview can directly use external virtual hard disk.

### DISCUSSION THREAD

**Discussion by YLinXin** [utmapp/UTM#3142] [UTM Help (macOS)]
**Describe the issue**
v2.2.4 cannot directly use virtual machines on my external SSD. When I open the virtual machines it copies the whole file to the internal SSD but it lacks so much storage to handle VM(100GB+).
I download and install the [VMware-Fusion-e.x.p-18656771_arm64.dmg](https://download3.vmware.com/software/fusion/file/VMware-Fusion-e.x.p-18656771_arm64.dmg). In the disk share mode it can use the virtual hard disk on my external SSD directly.

I'm waiting for it.

**Configuration**
* UTM Version: v2.2.4
* OS Version: 11.6
* Apple Silicon

**Comment by osy** [comment]
VMware has a market cap of $58 billion. We are a handful of developers working on our free time. Have patience.

Plus VMware isn’t a sandboxed app.
